server/src/main/java/io/whitefox/core/InternalTableName.java

```diff
@@ -0,0 +1,37 @@
+package io.whitefox.core;
+
+import io.whitefox.annotations.SkipCoverageGenerated;
+import java.util.Objects;
+
+public class InternalTableName {
```

Comment:
I am maybe a bit out of the loop since I wasnt working last week, but why do we need for each model also a class to hold the name?

Reply:
We don't *need* them but I was getting quite nervous when passing 4-5 strings as arguments like this https://github.com/agile-lab-dev/whitefox/blob/93d9748fbd233964b2078e90b7b700062bacee72/server/src/main/java/io/whitefox/core/services/ShareService.java#L82
given that java does not have named arguments it's easy to switch one for the other and end up with very hard to spot bugs :) see also: #112